internal/acceptance/job_test.go

```diff
@@ -110,6 +110,71 @@ func TestAccJobTasks(t *testing.T) {
 	})
 }
 
+func TestAccForEachTask(t *testing.T) {
+	t.Skip("Skipping this test because feature not enabled in Prod")
+	workspaceLevel(t, step{
+		Template: `
+		data "databricks_current_user" "me" {}
+		data "databricks_spark_version" "latest" {}
+		data "databricks_node_type" "smallest" {
+			local_disk = true
+		}
+
+		resource "databricks_notebook" "this" {
+			path     = "${data.databricks_current_user.me.home}/Terraform{var.RANDOM}"
+			language = "PYTHON"
+			content_base64 = base64encode(<<-EOT
+				# created from ${abspath(path.module)}
+				display(spark.range(10))
+				EOT
+			)
+		}
+
+		resource "databricks_job" "this" {
+			name = "{var.RANDOM}"
+
+			job_cluster {
+				job_cluster_key = "j"
+				new_cluster {
+					num_workers   = 20
+					spark_version = data.databricks_spark_version.latest.id
+					node_type_id  = data.databricks_node_type.smallest.id
+				}
+			}
+
+			task {
```

Comment:
`task` is part of API 2.0. Can we also have a test creating a multi task job? A job containing a `notebook_task` and a `for_each_task` should be fine.

Reply:
No, the `task` block is for API 2.1